checkin/models.py

```diff
@@ -3,6 +3,15 @@
 
 
 class Skill(models.Model):
+    """Skill with image
```

Comment:
Unless the docstring is a single linem start writing on the line after `"""`, that is:
```python
"""This is a description"""
```
and
```python
"""
This is a description

:var abc: Var description
"""
```
are fine, while
```python
"""This is a description

:var abc: Var description
"""
```
is not fine.